app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,20 @@
+class LikesController < ApplicationController
+  # def create
+  #  @post = Post.find(params[:id])
+  # @like = @post.likes.new
+  # @like.post_id = @post.id
+  # @like.author_id = @user.id
+  # @like.save
+  # redirect_to "/users/#{current_user.id}/posts/#{@post.id}"
+  # end
+  def create
+    @like = Like.new(like_params)
+    redirect_to user_post_path(id: @like.post_id, user_id: @like.author_id) if @like.save
+  end
```

Comment:
- [ ] @V0pe great work on your create method, however, it would be nice if you had conditional rendering for when the comment is created and when due to an error it's not created.

Reply:
@danmainah Thank, I believe I added the if statement at the end of 12. If not created, It wont do anything